tests/gateway-api.utils.js

```diff
@@ -133,7 +133,7 @@ function getMessageContents() {
 }
 
 function allMessageDocs() {
-  return testUtils.db.query('medic-client/messages_by_contact_date',
+  return testUtils.db.query('medic-client/messages_by_contact_date_test',
```

Comment:
I think this should just work because it includes docs and doesn't read the value - no code change necessary. Do the tests start failing?